docs/WebComponent.md

```diff
@@ -2,7 +2,7 @@
 
 To have the web component be able to use the same React components as the site application there needed to be a separate start script for the component. This required being able to customise how build was done and meant the `create-react-app` needed to be ejected. This copies configuration files into the project instead of running it all in create-react-app scripts.
 
-There is a custom webpack config file for the component `webpack.component.config.js` and a script in the `package.json`: `start:wc` which will start serving the web component.
+There is a custom webpack config file for the component `webpack.config.js` and a script in the `package.json`: `start:wc` which will start serving the web component.
```

Comment:
Also needs `start:wc -> start`

Reply:
Probably this file needs revising more extensively to be honest, because the paragraph above is also no longer correct